src/engine/cuecontrol.cpp

```diff
@@ -854,15 +832,15 @@ void CueControl::cueDefault(double v) {
 void CueControl::pause(double v) {
     QMutexLocker lock(&m_mutex);
     //qDebug() << "CueControl::pause()" << v;
-    if (v != 0.0) {
+    if (v != 0) {
```

Comment:
This falls in the category of try being smarter than the compiler. 

I have just checked this with gcc 6.2.0 32 bit: 

In the 0.0 case it produces exactly the same code if you use 
if (v != 0) or if (v != 0) or if (!v)
It uses "fldz" (FPU load zero) and "fucomip" (float compare) 

If you compare to an other literal value, lets say if (v != 15), a const double 15.0 can be found in the text segment and a float compare is used even if I enable optimization  -O3 

It seams that the compiler dos not like the "ficom" (float int compare) in this case  :-/ 

I should I revert this?